tools/cloud-build/daily-tests/create_deployment.sh

```diff
@@ -39,7 +41,12 @@ cd "$ROOT_DIR" ||
 		echo "*** ERROR: failed to access root directory ${ROOT_DIR} when creating blueprint"
 		exit 1
 	}
-make
+
+if [[ $ALWAYS_RECOMPILE != "no" || ! -f ghpc ]]; then
```

Comment:
nit: is it possible to pass `$ALWAYS_RECOMPILE` as a bool instead of a string?

Reply:
I gave thought to using a bash boolean but didn't want to spend time handling edge cases like invocation at shell (which shell?) and conversion of YAML value in playbook to bash environment.